README.md

```diff
@@ -1,9 +1,22 @@
 # eQ Survey Runner
 [![Build Status](https://travis-ci.org/ONSdigital/eq-survey-runner.svg?branch=master)](https://travis-ci.org/ONSdigital/eq-survey-runner) [![Code Issues](https://www.quantifiedcode.com/api/v1/project/1709e9d582cc479a86568a043117d4d0/badge.svg)](https://www.quantifiedcode.com/app/project/1709e9d582cc479a86568a043117d4d0) [![codecov](https://codecov.io/gh/ONSdigital/eq-survey-runner/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/eq-survey-runner) [![Dependency Status](https://gemnasium.com/badges/github.com/ONSdigital/eq-survey-runner.svg)](https://gemnasium.com/github.com/ONSdigital/eq-survey-runner)
 
-Based on python 3
+## Run with Docker
+
+To use docker to run eq-survey-runner you first need to build the container (this may take a few minutes on first run)
```

Comment:
From experience, there may be upstream changes that don't appear as changed to docker-compose until you don't use the cache - i.e the postgres image we're using is latest and could be updated, and similarly some of the npm deps aren't locked on specific version numbers. In those cases using cached layers doesn't correctly represent a new build. For those cases, it's worth being verbose in the docs on usage.